cypress/support/e2e.ts

```diff
@@ -5,8 +5,30 @@ import './commands';
 import { createGlobalOrganization, createGlobalProject } from './global-project';
 
 before(function () {
-  const devBaseUrlPort = Cypress.config().baseUrl?.split(':').slice(-1).toString();
-  if (devBaseUrlPort === '4101') {
+  // cy.log('ENV', Cypress.env());
+  // let splitLocalhost;
+  // // const devBaseUrlPort = Cypress.config().baseUrl?.split(':').slice(-1).toString();
+  // const baseUrl = Cypress.config().baseUrl;
+  // let localhost;
+  // // Assuming baseUrl = "https://localhost.com"
+  // if (baseUrl) {
+  //   const splitUrl = baseUrl.split('.');
+  //   if (splitUrl.length >= 4) {
+  //     localhost = splitUrl[splitUrl.length - 4];
+  //     cy.log('LOCALHOST', localhost); // Output: "localhost"
+  //   } else {
+  //     cy.log('The URL might not contain the expected segments.');
+  //   }
+  // } else {
+  //   cy.log('Base URL is not defined.');
+  // }
+  // if (localhost) {
+  //   splitLocalhost = localhost.split('/').slice(-1).toString();
+  // }
+
+  // if (devBaseUrlPort === '4101' || splitLocalhost !== 'localhost') {
+  if (Cypress.env('AWX_SERVER')) {
```

Comment:
I think AWX_SERVER is always defined that this code is trying to be run always.
That would fail the EDA E2E tests.